app/android/app/build.gradle

```diff
@@ -45,7 +45,7 @@ android {
         applicationId "com.rallyprotocol.flutter_example"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
-        minSdkVersion flutter.minSdkVersion
+        minSdkVersion 23
```

Comment:
We can do it here, but I think recommended practice in flutter is to use a `local.properties` file that overrides the flutter default, and then will be auto injected into the build script.